modules/persistence/tests/services/pages.test.ts

```diff
@@ -113,6 +113,16 @@ describe('pages module', () => {
           page_id: `${pagePrefix}/page1.txt`,
           filename: 'page1.txt',
           ast: { foo: 'foo', bar: { foo: 'baz' } },
+          facets: [
+            {
+              category: 'target_product',
+              value: 'atlas',
+              display_name: 'Atlas',
+              sub_facets: [
+                { category: 'sub_product', value: 'kubernetes-operator', display_name: 'Kubernetes Operator' },
+              ],
+            },
+          ],
```

Comment:
Similar to the above comment, I think it would be neat if we can make sure the facets are being updated properly when the facets themselves change without an AST change (assuming the facets are allowed to change without an AST change; if this is never possible, please ignore this comment!). Maybe a separate example where only the facets change?

Reply:
Great suggestion! added a test for this case where facets are updated/removed